ecs/daemon/inputs/task.conf

```diff
@@ -0,0 +1,14 @@
+[INPUT]
+    Name                tail
+    Tag                 app.*
+    Path                /var/lib/docker/containers/*/*.log
+    Exclude_Path        *container-cached*
+    multiline.parser    docker, cri
+    DB                  /var/fluent-bit/state/flb_container.db
+    Mem_Buf_Limit       ${MEM_BUF_LIMIT_TASK}
+    buffer_max_size     10M
+    Skip_Long_Lines     On
+    Refresh_Interval    10
+    Rotate_Wait         30
+    storage.type        filesystem
+    Read_From_Head      ${READ_FROM_HEAD_TASK}
```

Comment:
Is this an annoying comment? You may want to have a white space at the end of all your config files for consistency.